www/blog/reactive_social_skip_service_poc.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Reactive Social Network Service with Skip
+description: Build step by step a skip service
+slug: reactive_social_network_service_poc
+date: 2025-04-30
+authors: hubyrod
+---
+
+Reactive programming often sounds complex—but it doesn't have to be. What if you could actually see how data responds to change, in real time, right from your terminal?
+
+In this tutorial, we walk through building a small proof-of-concept social network backend using [Skip](https://www.skiplabs.io/) and TypeScript. It tracks users, groups, and their friendships—with automatic updates to "active friends" using Skip’s reactive computation graph.
+
+You’ll learn how to:
+
+- Set up a reactive Skip service
+- Define users and groups as live-updating resources
+- Connect it all to a REST API using Express
+- Observe real-time updates as the data evolves
+
+> "The simpler, the merrier" — this project keeps things minimal, focused, and easy to explore.
+
+From Alice adding a new friend to live data reactions, this guide makes reactive systems tangible.
+
+{/* truncate */}
+
+### The Simplest, the Merrier
+
+Ever heard the saying, "The simpler, the better"? Well, that especially holds true when it comes to learning—particularly in reactive programming. Think of it like what we expect from a social network: *reactivity* in response to change. Sure, a full-blown social network has more bells and whistles than a carnival, but here we're just tackling a small piece of it. This guide keeps things light and breezy, focusing on the essentials to show how to build a reactive system using **Skip**. It's a proof of concept.
+
+By keeping things simple and working right in the terminal, we can easily follow how data flows and how changes propagate—without getting lost in complexity. So let's dive in and have some fun while we're at it!
```

Comment:
```suggestion
By keeping things simple and working right in the terminal, we can easily follow how data flows and how changes propagate without getting lost in complexity.
```

Lots of em dashes that could be commas or no punctuation at all.

And deleting that last sentence is subjective preference, but personally I prefer less "fluff" like that.